ui/view/wallet/TransactionDetailsPopup.qml

```diff
@@ -142,6 +143,27 @@ CustomDialog {
             rowSpacing: 14
             columns: 2
             
+            SFText {
+                font.pixelSize: 14
+                color: Style.content_secondary
+                //% "Date"
+                text: qsTrId("tx-details-date-label") + ":"
+                visible: dateField.parent.visible
+            }
+            RowLayout {
+                visible: stm.state == "tx_info" && dialog.sendAddress.length && !(isIncome && isShieldedTx)
```

Comment:
why you hide this field? The date field available for all transaction types.

Reply:
Oh, you're a hawkeye. It seemed to me from the design that specifically for some transactions.

Reply:
Fixed.